allauth/socialaccount/providers/yandex/views.py

```diff
@@ -0,0 +1,31 @@
+import requests
+
+from allauth.socialaccount.providers.oauth2.views import (
+    OAuth2Adapter,
+    OAuth2CallbackView,
+    OAuth2LoginView,
+)
+
+from .provider import YandexProvider
+
+
+class YandexAuth2Adapter(OAuth2Adapter):
+    provider_id = YandexProvider.id
+    access_token_url = "https://oauth.yandex.ru/token"
+    authorize_url = "https://oauth.yandex.com/authorize"
```

Comment:
dot com there is not logical because of the most of yandex users have their accounts on dot ru. It is need to change it either to .ru, or add settings variable.

Reply:
This does not affect the functionality in any way, so it does not make any difference.

Reply:
This affects security because users cannot figure out how to log out of yandex.com because they are used to logging in and out of yandex.ru.
Secondly, users cannot use their current sessions at yandex.ru.
Everything else is very useful, thank you.